### ISSUE TITLE
ASP.NET passing of arguments does not work?

### ISSUE THREAD

**Issue by Slesa** [nordfjord/minimal-equinox#2]
I get the error 'Required parameter "id" was not provided from query string' when running any of these queries with curl.

When I changed the Guid to a string with manual parsing of the invoice id, the string parameter was always NULL. Even when I send 4711.
So my question is: does this example run as it is? This is very strange...

**Comment by bartelink** [comment]
I have never tried or looked, but are you basing your queries on ones in https://nordfjord.io/2022/12/05/equinox.html ? (Might be useful to show an example of what you're trying).
@nordfjord this repo's description should to link to your blog and/or the article ;)

**Comment by nordfjord** [comment]
Hey @Slesa Thanks so much for chiming in!

This seems to be the result of an over-eager refactor on my part.

```fsharp
let x = Func<int, string>(fun id -> string id)

// This is wildly different from the above
let toString (id: int) = string id
let toStringFunc = Func<_,_>(toString)
```

I did a refactor like this on the ASP.NET routes which led to this issue you're seeing. I'll open a PR and tag you in it.

Thanks again for chiming in 🙏

**Comment by nordfjord** [comment]
Closing since I've merged #3

**Comment by Slesa** [comment]
Great, thanks a lot.
